frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -250,28 +251,47 @@ export const createRoute = (
   return k8sCreate(RouteModel, route, dryRun ? dryRunOpt : {});
 };
 
-export const createResources = (
+export const createResources = async (
   formData: GitImportFormData,
   imageStream: K8sResourceKind,
   dryRun: boolean = false,
 ): Promise<K8sResourceKind[]> => {
   const {
+    application: { name: applicationName },
+    project: { name: projectName },
     route: { create: canCreateRoute },
     image: { ports },
   } = formData;
 
   const requests: Promise<K8sResourceKind>[] = [
-    createDeploymentConfig(formData, imageStream, dryRun),
     createImageStream(formData, imageStream, dryRun),
     createBuildConfig(formData, imageStream, dryRun),
   ];
 
-  if (!_.isEmpty(ports)) {
-    requests.push(createService(formData, imageStream, dryRun));
-    if (canCreateRoute) {
-      requests.push(createRoute(formData, imageStream, dryRun));
+  if (!formData.serverless.trigger) {
```

Comment:
I think this logic should bbe other way round. If serverless is enabled then do serverless stuff else just ccontinue with default flow.

Reply:
done